Dockerfile

```diff
@@ -0,0 +1,19 @@
+ARG PYTHON_VERSION=3.12
+
+FROM python:${PYTHON_VERSION} AS base
+WORKDIR /app
+COPY . ./
+
+FROM python:${PYTHON_VERSION}-slim AS stage
+WORKDIR /app
+ENV PYTHONUNBUFFERED=1
+COPY --from=base /app .
+RUN pip install --upgrade pip && \
+pip install -r requirements.txt
+
+RUN python manage.py migrate
+
+EXPOSE 8080
+
+ENTRYPOINT ["python", "manage.py", "runserver"]
+CMD ["0.0.0.0:8080"]
```

Comment:
Fix EOF (end-of-file)
<img width="450" alt="image" src="https://github.com/user-attachments/assets/aecbe597-099b-4765-a5a6-68ee636f44b4">

```suggestion
CMD ["0.0.0.0:8080"]

```

Reply:
add new line